packages/frontend/app/modifiers/update-document-title.ts

```diff
@@ -0,0 +1,26 @@
+import Modifier from 'ember-class-based-modifier';
+
+export default class UpdateDocumentTitle extends Modifier {
+  originalDocumentTitle: string;
+
+  constructor(owner, args) {
+    super(owner, args);
+    this.originalDocumentTitle = document.title;
+  }
+
+  get unreadMessageCount(): number {
```

Comment:
since these are only used in the one method, do they need to be defined as getters?

Reply:
All the examples in the docs have those as getters. I think it may have something to do with didReceiveArguments being updated whenever an argument's value is updated. I could be wrong though.

Reply:
There is no functional difference. It's just for readability. Since the docs include it as an example, I'm cool with it staying :)